Xamarin.Forms.Core/Shell/IShellContentController.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Xamarin.Forms
+﻿using System;
+
+namespace Xamarin.Forms
 {
 	public interface IShellContentController : IElementController
```

Comment:
should be hidden from intelisense

Reply:
This is not new , so why would we hide now?